doc/syslog/syslog-design.md

```diff
@@ -195,48 +236,76 @@ The `rsyslog-config` service performs the next actions:
+| welf_firewall_name    | severity | SYSLOG_CONFIG\|GLOBAL\|welf_firewall_name  |
+
+* severity: this field is a global log severity. 
+  this will be the default value for all the servers in the system
+  unless overridden by setting the `server.severity` field.
+  for example you can look at `2.4.3 configuration sample`
+  server "4.4.4.4" does not set `severity` field so it will use this global severity value.
+  server "4.4.4.5" does set it so it will use his local value and not this global value.
+  
+
+#### 2.3.2.2. Format: `set`:
+
+Format:
+template(name="WelfFormat" type="string" string="%TIMESTAMP% %HOSTNAME% id=firewall time=\"%timereported\
+:::date-year%-%timereported:::date-month%-%timereported:::date-day% %timereported:::date-hour%:%timereported:::date-minute%:%timereported\
+:::date-second%\" fw=\"{{ firewall-name  }}\" pri=%syslogpriority% msg=\"%syslogtag%%msg:::sp-if-no-1st-sp%%msg:::drop-last-lf%\"\n")
 
-###### Table 3: SSIP parameters
+### 2.3.3. SSIP parameters
 
-| SONiC  | Rsyslogd | Config DB Schema           |
-|:-------|:---------|:---------------------------|
-| key    | target   | SYSLOG_SERVER\|key         |
-| source | address  | SYSLOG_SERVER\|key\|source |
-| port   | port     | SYSLOG_SERVER\|key\|port   |
-| vrf    | device   | SYSLOG_SERVER\|key\|vrf    |
+**SSIP will have the next parameter mapping:**
+
+#### 2.3.3.1. Table 5: SSIP parameters
+| SONiC         | Rsyslogd | Config DB Schema                   |
+|:--------------|:---------|:-----------------------------------|
+| key           | target   | SYSLOG_SERVER\|key                 |
+| source        | address  | SYSLOG_SERVER\|key\|source         |
+| port          | port     | SYSLOG_SERVER\|key\|port           |
+| vrf           | device   | SYSLOG_SERVER\|key\|vrf            |
+| protocol      | protocol | SYSLOG_SERVER\|key\|protocol       |
+| filter_type   | ereregex | SYSLOG_SERVER\|key\|filter_type    |
+| filter_regex  | ereregex | SYSLOG_SERVER\|key\|filter_regex   |
+| severity      | priority | SYSLOG_SERVER\|key\|severity       |
 
-### 2.3.3 SSIP configuration
+### 2.3.4. SSIP configuration
 
 SSIP offers `vrf` and `source` parameters for flexible configuration management.  
 Each parameter combination requires a dedicated handling approach.
 
 **Note:**
 1. The destination might be not reachable over the specified `vrf`/`source`: no way to check - user's responsibility
 2. Additional validation is required when MGMT/DATA VRF is removed while reference still exists in syslog configuration
+3. `severity` field will be default log severity for rules that do not define it.
 
-#### 2.3.3.1 VRF/Source: `unset/unset`
+#### 2.3.4.1. VRF/Source: `unset/unset`
 
 Linux kernel decides which source IP to use within the default VRF.
 
 **Example:**
 ```
-*.* action(type="omfwd" target="2.2.2.2" protocol="udp")
+*.notice action(type="omfwd" target="2.2.2.2" protocol="udp")
```

Comment:
Current 'show runningconfiguration syslog' is matching lines starting with '*.*'. 
Will it break the show cli function? https://github.com/sonic-net/sonic-utilities/blob/master/show/main.py#L1687

Reply:
> Will it break the show cli function?

It will. I fixed it here: https://github.com/sonic-net/sonic-utilities/pull/2843